lightspeed/assemblies/assembly_setting-up-lightspeed.adoc

```diff
@@ -11,10 +11,12 @@ ifdef::context[:parent-context: {context}]
 
 [role="_abstract"]
 
-As a Red Hat customer portal administrator, you must configure {LightspeedShortName} to connect to your {ibmwatsonxcodeassistant} instance. This chapter provides information about configuring both{LightspeedShortName} cloud service and on-premise deployment.
+As a Red Hat customer portal administrator, you must configure {LightspeedShortName} to connect to your {ibmwatsonxcodeassistant} instance. This chapter provides information about configuring both {LightspeedShortName} cloud service and on-premise deployment.
```

Comment:
Second sentence should have an article "the" before the attribute, so it says "This chapter provides information about configuring both the RH Ansible Lightspeed cloud service and on-premise deployment."